main.py

```diff
@@ -3,14 +3,31 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    clock = pygame.time.Clock()
+    while True:
```

Comment:
**Suggestion:** Use a boolean flag to control the game loop exit condition for better readability.

**Existing Code:**
```Python
while True:
```

**Improved Code:**
```Python
running = True
while running:
    for event in pygame.event.get():
        if event.type == pygame.QUIT:
            running = False
```

**Details:**
Consider using a boolean flag to control the game loop exit condition for better readability and maintainability.